File: official/docs/node/webhooks/update_webhook.js
const Easypost = require('@easypost/api');

const api = new Easypost(process.env.EASYPOST_API_KEY);

const webhookId = api.Webhook.retrieve('hook_...').then(console.log);

api.Webhook.retrieve(webhookId).then((retrievedWebhook) => {
  retrievedWebhook.save().then(console.log);
});

File: official/docs/csharp/v3/webhooks/update-webhook.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

            Webhook webhook = await Webhook.Retrieve("hook_...");

            await webhook.Update();

            Console.WriteLine(JsonConvert.SerializeObject(webhook, Formatting.Indented));
        }
    }
}
